src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAO.scala

```diff
@@ -110,6 +110,10 @@ class ElasticSearchDAO(servers: Seq[Authority], indexName: String) extends Searc
     autocompleteSuggestions(client, indexName, criteria, groups)
   }
 
+  override def fieldSuggest(field: String, text: String): Future[Seq[String]] = {
```

Comment:
i just assumed it was required since i saw it on the other methods declared in SearchDAO but defined in this file